CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 ### 2.3.3 (TBD)
 
-- Feature: `telepresence intercept` now supports a `--to-pod` flag that can be used to port-forward sidecars' ports from an intercepted pod
+- Feature: Telepresence now supports installing the Traffic Manager
+  via Helm.  This will make it easy for operators to install and
+  configure the server-side components of Telepresence separately from
+  the CLI (which in turn allows for better separation of permissions).
+
+- Feature: As the traffic manager can now be installed in any
+  namespace via Helm, Telepresence can now be configured to look for
+  the traffic manager in a namespace other than `ambassador`.  This
```

Comment:
I think this usage should be `traffic-manager` bc we are talking about the service here.